File: Ohtap E Commerce Domain Models/Development/PHP/Objects/Payments/PaymentBasic.php
<?php
namespace Ohtap\ECommerce\DomainModels\Objects\Payments;



class PaymentBasic implements IPaymentData
{
    public $Currency;//Currency
    public $Types;//PaymentTypes
    public $Amount;
    public $Title;
    public $Name;
    public $Value;
    public $Description;
    public $State;
    public $OrderNo;
    public $ID;

}

File: Ohtap E Commerce Domain Models/Development/NetFramework/Models Objects/Models Objects/Payments/PaymentBasic.cs
using Ohtap.ECommerce.Domain.Models.Objects.Currencies;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Ohtap.ECommerce.Domain.Models.Objects.Payments
{
    public class PaymentBasic : IPaymentData
    {
        public virtual string ID { get; set; }
        public virtual string Code { get; set; }
        public virtual string Name { get; set; }
        public virtual int Order { get; set; }
        public virtual string Description { get; set; }
        public virtual int State { get; set; }
        public virtual string Title { get; set; }
        public virtual ICurrencyData Currency { get; set; }
        public virtual PaymentTypes Type { get; set; }
        public virtual decimal Amount { get; set; }
    }
}
